includes/class-wcpdf-settings-callbacks.php

```diff
@@ -53,6 +53,11 @@ public function checkbox( $args ) {
 
 		// output checkbox	
 		printf( '<input type="checkbox" id="%1$s" name="%2$s" value="%3$s" %4$s %5$s/>', $id, $setting_name, $value, checked( $value, $current, false ), !empty($disabled) ? 'disabled="disabled"' : '' );
+
+		// print store empty input if true
+		if( $store_unchecked ) {
+			printf( '<input type="hidden" name="wpo_wcpdf_setting_store_empty[]" value="%s"/>', $id );
```

Comment:
this doesn't work because the input `name` needs to be in the options array, so like this:
```php
printf( '<input type="hidden" name="%s[wpo_wcpdf_setting_store_empty][]" value="%s"/>', $option_name, $id );
```
otherwise it is not passed to the settings sanitization callback and still not saved as `0`...